test/EFCore.PG.Plugins.FunctionalTests/NodaTimeQueryNpgsqlTest.cs

```diff
@@ -377,15 +379,9 @@ public void GetCurrentInstant_from_Instance()
 
         NodaTimeContext CreateContext() => Fixture.CreateContext();
 
-        void AssertSql(params string[] expected)
-            => Fixture.TestSqlLoggerFactory.AssertBaseline(expected);
-
-        void AssertContainsSql(params string[] expected)
-            => Fixture.TestSqlLoggerFactory.AssertBaseline(expected, assertOrder: false);
-
         string Sql => Fixture.TestSqlLoggerFactory.Sql;
 
-        static Period DefaultPeriod;
+        static Period _defaultPeriod;
```

Comment:
In a different PR you have changed the same thing the back way. Personally I would like to follow .NET code style to use prefixes (`_` for instance fields, `s_` for static, and `t_` for thread static). @roji?

Reply:
Could you flag where that was? If it's inconsistent, I'd like to clean it up.

I believe the convention here (default from Rider/ReSharper) for private fields is to prefix with an underscore and lowercase, while all `static readonly` and `const` fields get special treatment of non-prefixed and uppercase.

In general, I prefer to stick with Rider/ReSharper defaults where possible, because it makes things easy when moving between open source and work projects. But that's just a personal preference.